### DISCUSSION TITLE
local installation problem

### DISCUSSION THREAD

**Discussion by mehrdadbn9** [apache/airflow#40112] [Q&A]
### What do you see as an issue?

https://airflow.apache.org/docs/apache-airflow/stable/start.html
base on upper url for installation would not got complete for debian base OS?

### Solving the problem

if is it possible I would change the env variable  like changing python version to Python 3 at the documentation and submit the pull request?

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by shahar1** [comment]
Welcome, and thank you for your post!
Unfortunately, I don't understand what issue you're trying to address and how you're willing to solve it.
As for what you stated in the issue, Apache Airflow should work fine on Debian `bookworm` (v12) using any of the suggested methods in the docs. As the issue is not clear, I don't see what env. variable needs changing at this point.
I'd be happy if you could provide more details about what you see as problematic, and maybe we could figure it out.

Converting this issue to a discussion.

**Comment by mehrdadbn9** [comment]
thanks all
